code/lib/cli/src/project_types.ts

```diff
@@ -1,4 +1,4 @@
-import { validRange, minVersion } from 'semver';
+import { minVersion, validRange } from 'semver';
```

Comment:
Do you have some kind of VSCode extension that alphabetically orders imports or are you doing this by hand?

Reply:
Haha, yes Webstorm does that automatically... I will put it off, makes PR a bit harder to review.